coopr-rest-client/pom.xml

```
@@ -12,11 +12,20 @@
     <version>0.9.9-SNAPSHOT</version>
   </parent>
 
+  <properties>
+    <commons.lang.version>2.5</commons.lang.version>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>co.cask</groupId>
       <artifactId>coopr</artifactId>
-      <version>0.9.9-SNAPSHOT</version>
+      <version>${project.parent.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
```

seems like the only thing that needed this was the not implemented yet exception.  If that's the case can we throw something else like an UnsupportedOperationException instead of adding another dependency?